tests/nnapi/specs/Ex/one_hot_2_ex_dynamic_nnfw.mod.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2018 The Android Open Source Project
+# Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# If depth or axis is from input, the output shape of one_hot cannot be decided at compilation time.
+# Therefore, its output is "dynamic tensor"
+#
+
+model = Model()
+
+indice_input = Input("input0", "TENSOR_INT32", "{2, 2}")
+depth = Int32Scalar("depth", 3)
+onvalue = Float32Scalar("on", 5.) # default value is 1.
+offvalue = Float32Scalar("off", 1.) # default value is 0.
+axis0 = Int32Scalar("axis", 0) # default value is -1.
+model_output0 = Output("output", "TENSOR_FLOAT32", "{3, 2, 2}")
+
+model0 = model.Operation("ONE_HOT_EX", indice_input, depth, onvalue, offvalue, axis0).To(model_output0)
```

Comment:
This doesn't look like dynamic test.

Reply:
It has removed,

Reply:
> See the implementation of onehot, If the axis(argument) was negative value, It assigns the rank of indices. but if indices is dynamic, the rank will be 0.

As I know as if indices is dynamic, we can get shape and rank of indices in `DynamicInferer::visit()`. Why do you think the rank will be 0?

Reply:
Well. I have checked that value using GDB. At that time the rank of indices is 0.
If axis > 0, axis then asix rank of indices(Now, 0).

In my think, It's good to move the axis resolving logic from OnehotLayer.configure to cker::onehot.

Could you please review again?

Reply:
I understand the problem now. Our runtime calls `OnehotLayer::configure()` in compilation time. And the runtime calls DynamicInferer::visit(OneHot) before `OnehotLayer::run()` in execution time. But `axis` was converted to positive number in `OnehotLayer::configure()` when dynamic indices's rank is 0.
Anyway, it looks good to move converting axis to `cker::OneHot()`.